src/main/java/com/endava/soa_5ed/task3_oop/Cat.java

```diff
@@ -0,0 +1,84 @@
+package com.endava.soa_5ed.task3_oop;
+
+public class Cat extends Feline implements Tail {
+    private double size;
+    private String name;
+
```

Comment:
You need just to add here @Sl4j that will create for you an object log.

Reply:
I've created a log4j class and used a Logger object in my Cat.java and Human.java classes. Is that okay?

Reply:
Yeap

Reply:
thank you